screens/ShopSearch.js

```diff
@@ -77,164 +77,174 @@ export default class ShopSearch extends React.Component {
-        <View style={styles.container}>
-          <OrangeBackground/>
-          <Toolbar navigation={this.props.navigation}/>
-          <View style={styles.mainView}>
-            <Text style={{color: '#ffffff', textAlign: 'center',fontFamily: 'Montserrat-SemiBold', fontSize: this.renderFontSize()}}>Let{`'`}s</Text>
-            <Text style={{color: '#ffffff',textAlign: 'center',fontFamily: 'Montserrat-SemiBold', fontSize: this.renderFontSize()}}>mailU!</Text>
-            <View style={styles.slantedBlueRectangle} />
-            <View style={styles.slantedPurpleRectangle} />
-            <View style={styles.whiteCard}>
-              <View style={styles.box}>
-                <View style={styles.rowBox}>
-                  <Text style={styles.searchText}>How big is your mail load?</Text>
-                </View>
-                <View style={{flexDirection: 'row'}}>
-                  <Text style={styles.quantityText}>{this.state.pickerSelectionSize}</Text>
-                  <View style={{position: 'absolute', marginTop: window.height*.05, marginLeft: window.width*.7}}>
-                    <DropDown selectedValue={this.state.pickerSelectionSize} setState={this.setPickerSize} items={sizeItems}/>
+  
+    static navigationOptions = {
+      header: null,
+      };
+      
+      letsgo = () => {
+        this.props.navigation.navigate('LoadingScreen')
+        }
+  
+      constructor(props) {
+        super(props);
+        this.state = {
+          fontLoaded: false,
+          comments: ' ', 
+          pickerSelectionSize: 'Choose Option',
+          pickerSelectionQuant: 'Choose Option',
+        }
+      }
+      
+      search = () => {
+        this.props.navigation.navigate('LoadingScreen')
+      }
+  
+      componentWillMount = () => {
+        FontLoad.then((res) => {
+          this.setState({ fontLoaded: true });
+        })
+      }
+      
+    render() {
+      return (
+        this.state.fontLoaded ? (
+          <View style={styles.container}>
+            <Toolbar navigation={this.props.navigation}/>
+            <View style={styles.mainView}>
+                <View style={styles.whiteBox}> 
+                  <View style={styles.rowBox}>
+                    <Text style={styles.searchText}>How big is your mail load?</Text>
                   </View>
+                  <Text style={styles.bigText}>{this.state.pickerSelectionSize}</Text>
+                  <View style={{marginLeft: window.width*.6, marginTop: -window.height*0.03}}>
+                    <DropDown selectedValue={this.state.pickerSelectionSize} setState={this.setPickerSize} items={sizeItems}/>
                   </View>
-                <View style={styles.searchLine}/>
-                <View>
-                <View style={styles.rowBox}>
-                  <Text style={styles.searchText2}>How many packages do you have?</Text>
+                  <View style={styles.searchLine}/>     
+                </View>
+                <View style={styles.whiteBox2}> 
```

Comment:
Remember you shouldn't have numbers in your variable names. You already made a `whiteBox` style. Just reuse it.